eth/downloader/downloader.go

```diff
@@ -1712,8 +1829,17 @@ func (d *Downloader) reportSnapSyncProgress(force bool) {
 		return
 	}
 	// Retrieve the current chain head and calculate the ETA
-	// We're going to cheat for non-merged networks, but that's fine
-	latest := d.pivotHeader
+	latest, _, _, err := d.skeleton.Bounds()
```

Comment:
skeleton sync is useless for bsc
that‘s why I remove them when big merge
bringing them in is ok, so we can pay lower effort when next pick ups